### DISCUSSION TITLE
QBKG11LM hold and release not working

### DISCUSSION THREAD

**Discussion by workinganimal** [Koenkk/zigbee2mqtt#5280] [Devices]
I have several Xiaomi QBKG11LM switches but i can´t get hold and release actions working. Single and double are working but hold and release aren´t. Am i doing something wrong or what do i have to do?

**Comment by workinganimal** [comment]
If i listen to the topic then I only get single and double but no hold and release. Any ideas what should i try or do?

**Reply by 9mateusz** [reply]
The same story. I do not know if i is mistake in docs or maybe I do not know the way how to trigger it. Holding button triggers 'single'.
legacy= false
Can anyone help?

**Comment by AlexeyHimki** [comment]
I have same question.
